.travis/debuild_test

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -ex
+
+# this script runs as root under docker 
+
+# Prepare for build
+export DEBIAN_FRONTEND=noninteractive
+export LC_ALL=C
+apt-get update
+apt-get -y dist-upgrade
+apt-get -y install devscripts build-essential lintian gdebi-core git
+
+DEB_VERS=$(LC_ALL=C dpkg-parsechangelog | grep '^Version:' | cut -d ' ' -f 2,2)
+PLAIN_VERS=$(echo ${DEB_VERS} | sed 's#-.*##')
+git archive --format=tar HEAD | gzip > ../singularity-container_${PLAIN_VERS}.orig.tar.gz
+
+# Install dependencies
+mk-build-deps
+mv singularity-container-build-deps_*.deb ..
+gdebi -n ../singularity-container-build-deps_*.deb
+
+# Build package
+debuild -us -uc
+
+echo "Directory containing the build .deb-files:"
+ls -la ..
+gdebi -n ../singularity-container_*.deb
+
+# Install python 3 and pylint
+apt-get -y install python3 python3-pip python3-setuptools
+pip3 install pylint
+
+# Install docker
+apt-get -y install apt-transport-https ca-certificates curl gnupg2 software-properties-common
+curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(lsb_release -cs) stable"
+apt-get update
+apt-get -y install docker-ce
+
+# run the test suite
+apt-get -y install libtool sudo e2fsprogs
+sed -i 's,^prefix=.*,prefix=/usr,' test.sh
+sed -i 's,^sysconfdir=.*,sysconfdir=/etc,' test.sh
+sed -i 's,^localstatedir=.*,localstatedir=/var,' test.sh
```

Comment:
```
prefix=
exec_prefix=
libexecdir=
sysconfdir=
localstatedir=
bindir=
```

Should all be set to what was used on the `./configure` line (`test.sh` gets generated from `test.sh.in`). There's hard coded paths that need to match up here because of the SUID bits.

Reply:
Oh yes! It seems this is not needed anymore. As of #1204, when I added those lines last year, `test.sh` was not correctly taking over what was passed to `configure`, but it seems that's corrected now. @c-ney : Just removing those `sed` lines should work :wink:.

Reply:
Now that you mention this, I remember having to put in an opensuse specific line in the past, but forgot to add it here and it still works. Will remove them from both `debuild_test` and `rpmbuild_test`

Reply:
Ahh! I remember why this was needed! It works fine on Debian, since `debuild`runs "in-place" and executes `configure` with full parameters. 
`rpmbuild`, though, runs in a sandbox, and calls `configure` there, and not where we actually started it. 
So either we have to steal `test.sh` from `/root/rpmbuild/BUILD/singularity-2.5.1.999/`, or replicate the parameters `rpmbuild` is using further up (i.e. before `make dist`) when `configure` is called to make `make dist` actually work. 
I'm unsure which is nicer...

Reply:
Right... and this is on the debian one... I'm working on a comment on the rpmbuild one. ;)